src/objects/config.ts

```diff
@@ -8,7 +8,6 @@ let config: RequiredGraphConfig | null = null
 
 const defaults: Omit<RequiredGraphConfig, 'runId' | 'fetch'> = {
   animationDuration: 500,
-  nodeRenderKey: (node) => `${node.id},${node.kind},${node.start_time},${node.end_time},${node.state_type},${node.label}`,
```

Comment:
Remove this config in favor if a more dynamic version in nodeContainerService that should work no matter what keys are used.